.github/workflows/code_testing.yml

```diff
@@ -36,9 +36,9 @@ jobs:
           sudo apt-get update -y
 
       - name: Install CMake
-        uses: lukka/get-cmake@v3.24.3
+        uses: lukka/get-cmake@v3.29.3
```

Comment:
Are you sure that this fixed the issue, I think we have it on `@latest` on some repos and I think the issue still occurs there